x-pack/plugins/fleet/server/services/package_policy.ts

```diff
@@ -391,6 +395,35 @@ class PackagePolicyService {
 
     return Promise.all(inputsPromises);
   }
+
+  public async runExternalCallbacks(
+    externalCallbackType: ExternalCallback[0],
+    schema: typeof NewPackagePolicySchema | typeof UpdatePackagePolicySchema,
+    newPackagePolicy: NewPackagePolicy,
+    context: RequestHandlerContext,
+    request: KibanaRequest
+  ): Promise<NewPackagePolicy> {
+    let newData = newPackagePolicy;
+
+    const externalCallbacks = appContextService.getExternalCallbacks(externalCallbackType);
+    if (externalCallbacks && externalCallbacks.size > 0) {
+      let updatedNewData: NewPackagePolicy = newData;
+
+      for (const callback of externalCallbacks) {
+        try {
+          updatedNewData = schema.validate(await callback(updatedNewData, context, request));
```

Comment:
can you move the `await callback` statement to it's own line. e.g. `const result = await callback(...)` or similar?